bin/deployer_keycloak

```diff
@@ -0,0 +1,249 @@
+    if "scope" in msg:
+        new_msg["optionalClientScopes"] = msg.pop("scope")
+        if "openid" in new_msg["optionalClientScopes"]:
+            new_msg["optionalClientScopes"].remove("openid")
+    if "grant_types" in msg:
+        for grant_type in msg["grant_types"]:
+            if grant_type == "authorization_code":
+                new_msg["standardFlowEnabled"] = True
+            if grant_type == "client_credentials":
+                new_msg["serviceAccountsEnabled"] = True
+            # TODO
+            # if grant_type == 'urn:ietf:params:oauth:grant-type:token-exchange':
+            #     new_msg['attributes']['oauth2.device.authorization.grant.enabled'] = True
+            if grant_type == "urn:ietf:params:oauth:grant-type:device_code":
+                new_msg["attributes"][
+                    "oauth2.device.authorization.grant.enabled"
+                ] = True
+            if grant_type == "implicit":
+                new_msg["implicitFlowEnabled"] = True
+    if "token_endpoint_auth_method" in msg:
+        if msg["token_endpoint_auth_method"] == "none":
+            new_msg["publicClient"] = True
+        new_msg["clientAuthenticatorType"] = map_token_endpoint_value(
+            msg.pop("token_endpoint_auth_method")
+        )
+    if "client_secret" in msg:
+        new_msg["secret"] = msg.pop("client_secret")
+    if "token_endpoint_auth_signing_alg" in msg:
+        new_msg["attributes"]["token.endpoint.auth.signing.alg"] = msg.pop(
+            "token_endpoint_auth_signing_alg"
+        )
+    if "jwks" in msg:
+        new_msg["attributes"]["use.jwks.string"] = True
+        new_msg["attributes"]["jwks.string"] = msg.pop("jwks")
+    if "jwks_uri" in msg:
+        new_msg["attributes"]["use.jwks.url"] = True
+        new_msg["attributes"]["jwks.url"] = msg.pop("jwks_uri")
+    if "refresh_token_validity_seconds" in msg:
+        new_msg["attributes"]["client.offline.session.idle.timeout"] = msg[
+            "refresh_token_validity_seconds"
+        ]
+        new_msg["attributes"]["client.offline.session.max.lifespan"] = msg.pop(
+            "refresh_token_validity_seconds"
+        )
+    if "code_challenge_method" in msg:
+        new_msg["attributes"]["pkce.code.challenge.method"] = msg.pop(
+            "code_challenge_method"
+        )
+    if "access_token_validity_seconds" in msg:
+        if msg["access_token_validity_seconds"] < 60:
+            new_msg["attributes"]["access.token.lifespan"] = 60
+        else:
+            new_msg["attributes"]["access.token.lifespan"] = msg.pop(
+                "access_token_validity_seconds"
+            )
+    # TODO
+    # if 'id_token_timeout_seconds' in msg:
+    #     new_msg['attributes']['id.token.lifespan'] = msg.pop('id_token_timeout_seconds')
+    if "device_code_validity_seconds" in msg:
+        new_msg["attributes"]["oauth2.device.code.lifespan"] = msg.pop(
+            "device_code_validity_seconds"
+        )
+    print(json.dumps(new_msg))
```

Comment:
Use logger insted of printing messages to stdout

Reply:
This `print()` was added for development purposes. I have removed this line